### DISCUSSION TITLE
Import passwords?

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1343] [Ideas]
There is an export feature available for passwords that gives a csv file. Would be great if we could import passwords in the same file format.

**Comment by anuraganand-15** [comment]
Go to chrome://flags and enable password import flag.

**Comment by nvnth** [comment]
I did.i still don't see an import option or am I suppose to do something else?
![Screenshot_2022-07-01-19-19-27-192_org bromite bromite](https://user-images.githubusercontent.com/87908095/176907755-0db04b4b-2925-4f0b-b59a-c5ae88c7ac10.jpg)

**Comment by Padfoot-bag** [comment]
> There is an export feature available for passwords that gives a csv file. Would be great if we could import passwords in the same file format.

I have tried this method and it's not working even after toggling the feature enabled/disabled. Is there any other way for import passwords?

**Reply by uazo** [reply]
> I have tried this method and it's not working even after toggling the feature enabled/disabled

that flag doesn't work on android, all code is [disabled](https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/passwords/settings/password_manager_porter.cc;l=33;drc=74e6b58d295dae0f6445e29f8ef0f1cb1d841684;bpv=1;bpt=1)
I suggest you use a program like keepass because they are not encrypted but simply obfuscated on disk.
